tarodb/asset_minting.go

```diff
@@ -130,7 +135,7 @@ type PendingAssetStore interface {
 	// FetchSeedlingsForBatch is used to fetch all the seedlings by the key
 	// of the batch they're included in.
 	FetchSeedlingsForBatch(ctx context.Context,
-		rawKey []byte) ([]AssetSeedling, error)
+		rawKey []byte) ([]sqlc.FetchSeedlingsForBatchRow, error)
```

Comment:
I think we can avoid these intermediate structs by modifying the seedling model to align with the queries and vice versa - not strictly required though.

Reply:
Usually we type def these, but I left it in for now. Happy to change though.